database/migrations/default/1728059618657_signal_eng_areas/up.sql

```diff
@@ -0,0 +1,23 @@
+alter table non_coa_roadways drop column id,
+add column updated_at timestamptz default now() not null;
```

Comment:
Because this layer has just one feature and it will always need to be deleted and replaced in order to update it, i don't think it makes sense to furnish this table with a serial `id` column.

Reply:
i think these changes are good :+1:

Reply:
tbf though wont all the geospatial layers always have the same `updated_at` and `created_at` values? so in that case do we really need `updated_at`?

Reply:
I guess having both updated at and created at on there and the same value really hammers home that its a delete and replace layer

Reply:
i think my point was moot after realizing the answer to my own question here https://github.com/cityofaustin/vision-zero/pull/1571#discussion_r1792353907

it actually will be possible for `updated_at` and `created_at` to be different bc this script does upserts and doesnt wipe every row from the table on each refresh

Reply:
just noting that this helped me test, and i saw the `updated_at` timestamps update when running the signal engineer areas script a second time 👍